db/migrate/20220205123027_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
Please add `post_id` as a foreign key and its index as well. This way a comment would be associated with a post. Moreover, please add `author_id` as a foreign key instead of `user_id`.